______________________________________________________________
Stack[0x603000000048] called void StackPush(int, Stack*, int, const char*)(10) from src/main.cpp
{
   size = 30;
   capacity = 4;
   stk[0x603000000048]
   {
   }
}
ABORT
ERROR_ARRAY_EXIT

ABORT
ERROR_HASH_STRUCT

______________________________________________________________
Stack[0x603000000048] called void StackPush(int, Stack*, int, const char*)(11) from src/main.cpp
{
   size = 30;
   capacity = 4;
   stk[0x603000000048]
   {
   }
}
ABORT
ERROR_ARRAY_EXIT

ABORT
ERROR_HASH_STRUCT

______________________________________________________________
Stack[0x603000000048] called void StackPush(int, Stack*, int, const char*)(12) from src/main.cpp
{
   size = 30;
   capacity = 4;
   stk[0x603000000048]
   {
   }
}
ABORT
ERROR_ARRAY_EXIT

ABORT
ERROR_HASH_STRUCT

______________________________________________________________
Stack[0x603000000048] called void StackPush(int, Stack*, int, const char*)(13) from src/main.cpp
{
   size = 30;
   capacity = 4;
   stk[0x603000000048]
   {
   }
}
ABORT
ERROR_ARRAY_EXIT

ABORT
ERROR_HASH_STRUCT

